FirebaseABTesting/Sources/ABTExperimentPayload.m

```diff
@@ -0,0 +1,151 @@
+// Copyright 2020 Google
```

Comment:
Copyright 2020 Google LLC

New copyrights should follow go/copyright

Reply:
Fixed in 511e0b3.